### PR TITLE
Doc 8793

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

**Comment by tonyjhillman** [comment]
Comments that have driven these changes are listed and responded to at https://github.com/couchbase/docs-server/pull/2024.

File: modules/install/pages/upgrade-cluster-offline.adoc [hunk 1]
```diff
@@ -51,18 +52,17 @@ cbbackupmgr backup --archive ${ABS_PATH_TO_ARCHIVE} --repo ${REPO_NAME} --cluste
...
 +
 For the equivalent procedure as performed by the Backup Service, see xref:manage:manage-backup-and-restore/manage-backup-and-restore.adoc#run-an-immediate-backup[Run an Immediate Backup].
-+
-In the event of node-upgrade failing, the backed-up data can be restored by means of xref:backup-restore:cbbackupmgr-restore.adoc[cbbackupmgr restore] or xref:cli:cbtools/cbrecovery.adoc[cbrecovery]; or by means of the Backup Service (see xref:manage:manage-backup-and-restore/manage-backup-and-restore.adoc#restore-backups[Restore Backups]).
 
 . Disable auto-failover (to prevent auto-failover from occuring when individual nodes stop communicating with the rest of the cluster, during their upgrade).
-Disablement is performed by means of the xref:manage:manage-settings/general-settings.adoc[General Settings] screen, by means of the xref:manage:manage-settings/general-settings.adoc#node-availability[Node Availability] panel.
-This needs to be done only once, on one node.
+Disablement is performed by means of Couchbase-Server xref:manage:manage-settings/general-settings.adoc[General Settings]; and can be performed by means of the UI, the CLI, or the REST API.
```
Comment (jamesl33) [inline modules/install/pages/upgrade-cluster-offline.adoc]: Not sure this reads quite right, what does 'by means of "Couchbase-Server" refer to here?

Nitpicking, but you've used "by means of" three times here:
1) By means of Couchbase-Server
2) and can be performed by means of the UI, the CLI or the REST API
3) For example, to disable auto-failure by means of the CLI

Comment (tonyjhillman) [reply modules/install/pages/upgrade-cluster-offline.adoc]: Couchbase-Server General Settings. General Settings is the title given to all the related settings in the UI. It's also the title of the page that hosts all the examples - for UI, CLI, and REST. 

Well, I think the repetition is okay. Sometimes it is actually less distracting to be a bit mechanical with syntax. But I can, maybe, put "specifically be performed" in, before listing out the interfaces. Or something like that.

Comment (tonyjhillman) [reply modules/install/pages/upgrade-cluster-offline.adoc]: Disablement is performed by means of Couchbase-Server General Settings, using the UI, the CLI, or the REST API. Disablement needs to be performed only once, on one node.

For example, to disable auto-failover by means of the CLI, enter a command of the following form:

Comment (jamesl33) [reply modules/install/pages/upgrade-cluster-offline.adoc]: Oh apologies, I missed the 'General Settings' but this time around.

**Review by jamesl33 — APPROVED** [review]

**Pull request merged**